test/classes/initializers/where-clause4.good

```diff
@@ -0,0 +1,2 @@
+where-clause4.chpl:12: error: unresolved call '_new(type Foo, 3.4)'
```

Comment:
Yeah.  I'm also tracking the fact that we get a whole lot of extra output when the problem is with the init function itself - this is likely to impact us increasingly as we switch library types to use the new style initializers, so we'll probably want to clamp down on what we output in those cases.